lib/trailblazer/operation/validate.rb

```diff
@@ -22,7 +22,7 @@ def self.Validate(skip_extract: false, name: "default", representer: false, key:
         options = {task: activity, id: "contract.#{name}.validate", outputs: activity.outputs}
 
         # Deviate End.extract_failure to the standard failure track as a default. This can be changed from the user side.
-        options = options.merge(Activity::DSL.Output(:extract_failure) => Activity::DSL.Track(:failure)) unless skip_extract
+        options = options.merge(Activity::DSL.Output(:extract_failure) => Activity::DSL.Track(:failure)) unless skip_extract || representer
```

Comment:
we add the extract step only `unless skip_extract || representer` so makes sense to have the same block here